src/OpenLoco/Map/TileLoop.hpp

```diff
@@ -39,8 +39,8 @@ namespace OpenLoco::Map
         class Iterator
         {
         private:
-            const TilePos2& _bottomLeft;
-            const TilePos2& _topRight;
+            const TilePos2 _bottomLeft;
+            const TilePos2 _topRight;
```

Comment:
Why are these no longer references? I guess because the ptr size is equal to the TilePos2 struct size?

Reply:
I got scared whilst debugging as the end iterator points to invalid memory for these. I think it isn't an issue but I'm not entirely sure. So on the safe side changed it